src/hardware/encoder.cpp

```diff
@@ -83,22 +84,22 @@ uint16_t readEncoderRegister(uint16_t registerAddress) {
     // Deselect encoder
     digitalWrite(ENCODER_CS_PIN, HIGH);
 
-    // Return angle value as uint16
+    // Return value as uint16
     return rxbuf[0] << 8 | rxbuf[1];
 }
 
 
 // Read multiple registers
 // ! Flat out doesn't work
-void readMultipleEncoderRegisters(uint16_t registerAddress, uint16_t data[]) {
+void readMultipleEncoderRegisters(uint16_t registerAddress, uint16_t* data, uint16_t dataLength) {
```

Comment:
We have to pass array length because `sizeof()` only works if the array size is known to the compiler.

Reply:
Good to know for future